dockerfile/cuda11.1.1.dockerfile

```diff
@@ -128,6 +128,6 @@ ADD third_party third_party
 RUN make -C third_party cuda
 
 ADD . .
-RUN python3 -m pip install .[nvidia,torch,ort] && \
+RUN python3 -m pip install .[nvidia,torch,ort,mpi4py] && \
```

Comment:
pls update the pip install in cuda pipeline and all dockerfiles

Reply:
updated